core/controllers/admin.py

```diff
@@ -202,6 +202,15 @@ def post(self):
             raise
 
     def _reload_exploration(self, exploration_id):
+        """Reloads the exploration in production corresponding to the given
```

Comment:
Ah... ok. I was confused because of the exception error message in [this](https://github.com/oppia/oppia/pull/6041/files#diff-1a1590952ff79dbae57a5ba1b3123d9cL213) line

Reply:
Done